fe/fe-core/src/main/java/com/starrocks/connector/Connector.java

```diff
@@ -24,4 +24,8 @@ default void shutdown() {}
      * For example, hive connector must require "hive.metastore.uris" property.
      */
     default void validate() {}
+
+    default void onCreate() {}
```

Comment:
why do we need `onCreate` and `onDrop`? is constructor and `shutdown` not enough?